docs/products/access/api-reference/description-oauth-endpoint.md

```diff
@@ -245,3 +245,184 @@ See the OAuth specification for [token revocation](https://tools.ietf.org/html/r
+
+### Introduction
+
+The OAuth Device Flow is an extension to the standard OAuth 2.0 protocol, specifically designed for devices with no browser or limited input
+capability. This flow enables users to authorize such devices in a user-friendly and secure manner. After entering the correct code, the
+user proceeds through the standard authorization process, including authentication and, if enabled on the client, consent. For detailed
+specifications, refer to [RFC 8628](https://tools.ietf.org/html/rfc8628). The following diagram illustrates the flow:
+
+```mermaid
+sequenceDiagram
+    autonumber
+    participant Client as Device Client
+    participant Access
+    participant User
+    activate Client
+    Client ->> Access: Device Authorization Request
+    activate Access
+    Access -->> Client: Device Code, User Code and Verification URI
+    deactivate Access
+    Client ->> User: Display User Code & Verification URI
+    activate User
+
+    par
+        User ->> Access: Enter Code on Webpage
+        activate Access
+        Access -->> User: Redirect to authorization flow
+        deactivate Access
+        User ->> Access: Authorize Client
+        activate Access
+        Access -->> User: Status page
+        deactivate Access
+        deactivate User
+    and
+        loop Client not authorized yet
+            Client ->> Access: Token Request
+            activate Access
+            alt not authorized yet
+                Access -->> Client: Pending Authorization
+                Client ->> Client: Wait 5 seconds
+            else already authorized
+                Access -->> Client: Access Token
+            end
+            deactivate Access
+        end
+    end
+    deactivate Client
+```
+
+1. **Device Authorization Request**
+    - The *Device Client* initiates the process by sending a request to the *Access*.
+2. **Access responds with Device Code, User Code and Verification URI**
+    - The *Access* responds to the *Device Client* with a *Device Code* and a User Code.
+    - The *Device Code* will be used by *Device Client* to generate the Access Token.
+    - The User Code will be presented to the *User*.
+    - Verification URI will be used by the *User* to authorize the device.
+3. **Display User Code & Verification URI**
+    - The *Device Client* displays the User Code to the user and Verification URI (as text or QR code).
+    - The Verification URI points to a webpage hosted by the *Access* and its theme can be customized
+      using [User Code Template](../appendix/templates/templates.md#user-code-template)
+4. **User Enters Code on Webpage**
+    - The *User* uses another device (e.g., a smartphone) to visit the webpage (as instructed by the *Device Client*) and enters the *User
+      Code* or confirms it if already entered.
+5. **Redirect to Authorization Flow**
+    - After successful validation of the User Code, the *Access* component redirects the *User* to an authorization flow that is the same
```

Comment:
```suggestion
    - After successful validation of the User Code, the *Access* component redirects the *User* to the authorization flow of the configured default IdP for this Web Client.
```

Reply:
it does not need to be the default IdP - you can select IdP in Device Authorization Request